src/utils/response-middleware.js

```diff
@@ -9,10 +9,14 @@ import _ from "lodash";
 export default function responseMiddlewareFactory() {
   return function responseMiddleware(result, req, res, next) {
     if (_.isError(result)) {
+      const errorData = {
+        error: (result.stack || result),
+        req: _.pick(req, "url", "method")
```

Comment:
If it's a javascript error, we should also try to get the `message` property here.